examples/PromptInjectionDetector/PromptInjectionGPT4o.ipynb

```diff
@@ -0,0 +1,439 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import braintrust\n",
+    "from autoevals import NumericDiff\n",
+    "from autoevals import Levenshtein \n",
+    "\n",
+    "braintrust.login(api_key=os.environ.get(\"BRAINTRUST_API_KEY\", \"<Your Braintrust API Key Here>\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "id": "Pd75YlQvw1K8"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Experiment numericdiff_prompt_injection-1716236164 is running at https://www.braintrust.dev/app/braintrustdata.com/p/Prompt%20Injection%20Detection/experiments/numericdiff_prompt_injection-1716236164\n",
+      "Prompt Injection Detection (data): 30it [00:00, 42366.71it/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "f58139264a7d49708c5fea62becb5aad",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Prompt Injection Detection (tasks):   0%|          | 0/30 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "=========================SUMMARY=========================\n",
+      "90.00% 'NumericDiff' score\n",
+      "\n",
+      "0.90s duration\n",
+      "\n",
+      "See results for numericdiff_prompt_injection-1716236164 at https://www.braintrust.dev/app/braintrustdata.com/p/Prompt%20Injection%20Detection/experiments/numericdiff_prompt_injection-1716236164\n"
+     ]
+    }
+   ],
+   "source": [
+    "eval_result = await braintrust.Eval(\n",
+    "    \"Prompt Injection Detection\",\n",
+    "    data=prompts_and_labels,\n",
+    "    task=classify_prompt,\n",
+    "    scores=[NumericDiff],\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "bAQ3gHsdh3JU"
+   },
+   "source": [
+    "## Quickly understand results with Braintrust's Experiment\n",
+    "\n",
+    "The cell above will print a link to the BrainTrust experiment. Click on it to investigate where we can improve our AI app.\n",
+    "\n",
+    "Looking at our results table (in the screenshot below), we can focus on the \"misses\". It looks like we primarily have a false positive problem: The model is labeling a prompt as an injection attempt when it should not be. We have one single false negative problem. \n",
+    "\n",
+    "There are two followups we can decide to take from this information.\n",
```

Comment:
It'd be helpful to write what we expect to happen if the hypotheses are correct, i.e. that we'd expect the diff score to increase. Although, I may just be saying that because we didn't really talk about what the score was before this point. Maybe that would help!

Reply:
Great point, added overview of score _and_ that we'd expect the score to increase